docs/services/navigation/orientation.md

```diff
@@ -0,0 +1,100 @@
+---
+title: "How Orientation is Measured in Viam"
+linkTitle: "Orientation"
+weight: 10
+type: "docs"
+description: "How Viam reads and utilizes the orientation measurements reported by some models of movement sensor."
+aliases:
+    - "/appendix/orientation-vector"
+    - "/internals/orientation-vector"
+---
+
+How Viam's platform reads and utilizes the orientation measurements reported as `Readings` by the following {{< glossary_tooltip term_id="model" text="models" >}} of [movement sensor](/components/movement-sensor):
+
+- [imu-wit](/components/movement-sensor/imu/imu-wit/)
+- [imu-vectornav](/components/movement-sensor/imu/imu-vectornav/)
+
+An `Orientation` reading specifies the orientation of an object in 3D space as an "orientation vector", or, its position within the [cartesian coordinate system](https://en.wikipedia.org/wiki/Cartesian_coordinate_system) relative to some specific `origin` point that you, the user, need to choose and configure for your robot.
+
+## Client side
+
+You compose "orientation vectors" following the same protocol to specify relative orientations of components when using the [Motion Service](../../services/motion/) and [Frame System](../../services/frame-system/).
+
+An example of an `Orientation` reading or, an orientation vector:
```

Comment:
This shows output but not how a user would input an orientation vector to, say, the motion service.

Reply:
Does leaving the orientation vector page up satisfy this requirement?